src/main/java/rx/Notification.java

```diff
@@ -67,8 +67,10 @@
      *
      * @param <T> the actual value type held by the Notification
      * @param type
+     * @deprecated this method does the same as {@link #createOnCompleted()} and does not use the passed in type hence it's useless.
```

Comment:
The argument allows for explicit type inference without needing to specify it on the method call directly when you already have a typed `Class`.